Firestore/Source/API/FIRFirestore.mm

```diff
@@ -97,6 +97,8 @@ + (void)initialize {
   if (self == [FIRFirestore class]) {
     SetThrowHandler(ObjcThrowHandler);
   }
+
+  Firestore::SetClientLanguageHeader("gl-objc/2.0");
```

Comment:
This value was previously `"gl-objc/"` because the version doesn't matter/isn't maintained. Changing this needlessly breaks with existing data. Unless we're specifically going to search for/audit existing usage of this in sawmill I think we're better off leaving the version out (which makes this a no-op as far as downstream effects are concerned).

Reply:
Ok, makes sense. In the future, I'd like us to consider logging the LLVM version instead because, AFAIU, that's what actually determines features available to the developer.

Reply:
Future work for llvm version sg. The token still has a 2.0 in it that should be removed.

Reply:
Oh, sorry about that, fixed now.